internal/cli/root/builder.go

```diff
@@ -65,7 +67,17 @@ no additional shell configuration is necessary, see https://docs.brew.sh/Shell-C
 // This is important in particular for Atlas as it dynamically sets flags for cluster creation and
 // this can be slow to timeout on environments with limited internet access (Ops Manager)
 func Builder(profile *string, argsWithoutProg []string) *cobra.Command {
-	rootCmd = cli.Builder()
+	rootCmd := &cobra.Command{
```

Comment:
moving this in from the `cli.go` file they were both serving the same kind of functionality